src/Yubikey/Validate.php

```diff
@@ -269,7 +269,9 @@ public function check($otp, $multi = false)
 
         $clientId = $this->getClientId();
         if ($clientId === null) {
-            throw new \InvalidArgumentException('Client ID cannot be null');
+            if (!$this->getUseSecure())
```

Comment:
Yes, if you want, :).

On 18/03/16 12:30, Chris Cornutt wrote:
>
> In src/Yubikey/Validate.php
> <https://github.com/enygma/yubikey/pull/20#discussion_r56643355>:
>
> > @@ -269,7 +269,9 @@ public function check($otp, $multi = false)
> >  
> >          $clientId = $this->getClientId();
> >          if ($clientId === null) {
> > -            throw new \InvalidArgumentException('Client ID cannot be null');
> > +            if (!$this->getUseSecure())
>
> Can you add the |{| and |}| on this for consistency?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/enygma/yubikey/pull/20/files/67816d0b4177845c29a2370c7aa62fc3fbc95997#r56643355>
>